queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -78,6 +75,11 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
     if business.legal_type == Business.LegalTypes.COOP:
         _update_cooperative(dissolution_filing, business, filing_rec, dissolution_type)
 
+    with suppress(IndexError, KeyError, TypeError):
+        filing_meta.dissolution = {**filing_meta.dissolution,
+                                   **{'dissolutionType': dissolution_type},
+                                   **{'dissolutionDate': datetime.date(filing_rec.effective_date).isoformat()}}
```

Comment:
dissolution date is different from effective date for voluntary dissolution I think

Reply:
We can fix this in a follow up PR because there might be a gap in setting the dissolution date in the business too.